### DISCUSSION TITLE
Spinner on Component Overview page is not an anchor

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1963] [General] [answered]
Hey team, looks like the Spinner anchor is just text on the Component Overview page, but using the sidebar to navigate to it works fine.

https://paste.twilio.design/components
https://paste.twilio.design/components/spinner

![image](https://user-images.githubusercontent.com/6936736/138331857-699186de-99b3-41bf-b92e-aaed8b010d5f.png)

**Comment by loreina** [comment, marked as answer]
Hey @vnguyen94, thanks for catching this! I just updated the doc site so the table should now link out to the Spinner component page. :)

**Reply by vankhoawin** [reply]
quick fix 🏃  looks good!
